lib/reporters/html.js

```diff
@@ -127,6 +145,10 @@ function HTML(runner, options) {
 
   runner.on(EVENT_SUITE_END, function (suite) {
     if (suite.root) {
+      if (stats.failures === 0) {
+        text(resultIndicator, '✓');
```

Comment:
Note that this will show a check even if all tests are skipped, I hope that's OK, I see we have a skipped symbol but I didn't know if supporting that at root level was desired / in-scope for this PR, so I left it like this